.gitignore

```diff
@@ -15,6 +15,10 @@ node_modules
 .env.development.local
 .env.test.local
 .env.production.local
+*.iml
+.idea
+config.yaml
```

Comment:
This can be removed. We have a `/config.yaml` rule to ignore a config file in the root directory and this would also ignore some config files in other directories.

```suggestion
```